iac-extensions/docker/src/main/java/org/sonar/iac/docker/parser/DockerNodeBuilder.java

```diff
@@ -76,16 +78,57 @@ public Object createTerminal(Input input, int startIndex, int endIndex, List<Tri
     return new SyntaxTokenImpl(value, range, getCommentsForToken(range));
   }
 
+  /**
+   * Compute the {@link TextRange} from a value at the given {@code startIndex}, regarding the provided {@code input} (source code).
+   * In case it extend to multiple lines, we build a {@link CompoundTextRange} object with a reference to each line {@link TextRange}.
+   * It is used to track back tokens in HereDoc, as currently the full HereDoc content of an instruction is considered as a single {@link SyntaxToken},
+   * before it is being parsed by a specific HereDoc parser which then call this tokenRange method to split into multiple {@link SyntaxToken} with
+   * correct range.
+   * The {@code startIndex} is the position on the instruction line where the provided value begin.
```

Comment:
```suggestion
   * <p/>   
   * The {@code startIndex} is the position on the instruction line where the provided value begin.
```

Reply:
Did you mean to add a line break?
I'm not sure how valid `<p/>` is, in pure HTML I never saw this usage and from what I read it should be `<br />`.